src/sections/chat/ChatBoxSectionNew.tsx

```diff
@@ -1,12 +1,60 @@
-import React from "react";
+import { ItemVV2 } from "components/reusables/SharedStylingV2";
+import { device } from "config/Globals";
+import { ChatLocalContext } from "contexts/ChatLocalContextNew";
+import React, { useContext } from "react";
+import styled from "styled-components";
 import { ChatConversationSectionNew } from "./ChatConversationSectionNew";
 import { ChatWelcomeSectionNew } from "./ChatWelcomeSectionNew";
 
 export const ChatBoxSectionNew = () => {
+
+  const { selectedChat } = useContext(ChatLocalContext)
+
   return (
-    <>
-      <ChatWelcomeSectionNew />
-      <ChatConversationSectionNew />  
-    </>
+    <Box chatActive={selectedChat}>
+      <Container>
```

Comment:
We can use either container or box, both are not needed here.Better we keep the sidebar and chatbox section structure similar, so We can keep Container

Reply:
Add types too like mentioned in chatSideBArSection review comment

Reply:
removed box